cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go

```diff
@@ -37,6 +37,53 @@ const (
 	NodesKubeadmConfigClusterRoleName = "kubeadm:nodes-kubeadm-config"
 )
 
+// UploadRemoveConfiguration uploads a new configuration with the client removed
+func UploadRemoveConfiguration(cfg *kubeadmapi.InitConfiguration, client clientset.Interface) error {
+	fmt.Printf("[reset] removing the configuration stored in the ConfigMap %q in the %q Namespace\n", kubeadmconstants.KubeadmConfigConfigMap, metav1.NamespaceSystem)
```

Comment:
What about "removing node info from the kubeadm-config config map"?

Reply:
+1